### DISCUSSION TITLE
What's Jekyll? · Alan Plotko

### DISCUSSION THREAD

**Discussion by alanplotko-com-giscus** [alanplotko/alanplotko.github.io#62] [Blog Post Discussions]
# What's Jekyll? · Alan Plotko

Alan Plotko is a Software Engineer who loves building new things.

http://localhost:4000/blog/whats-jekyll

**Comment by alanplotko** [comment]
This is a test comment!

**Reply by alanplotko** [reply]
:+1:

**Comment by alanplotko** [comment]
This reply is made directly through the discussions board.
